src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3790,6 +3790,71 @@ internal sealed partial class LazyCoalesceExpression : BaseCoalesceExpression, I
         public override IOperation WhenNull => SetParentOperation(_lazyWhenNull.Value, this);
     }
 
+    internal abstract partial class BaseCoaleseAssignmentOperation : Operation, ICoalesceAssignmentOperation
+    {
+        protected BaseCoaleseAssignmentOperation(bool isChecked, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(OperationKind.CoalesceAssignment, semanticModel, syntax, type, constantValue, isImplicit)
+        {
+            IsChecked = isChecked;
+        }
+
+        public abstract IOperation Target { get; }
+        public abstract IOperation WhenNull { get; }
+        public bool IsChecked { get; }
+
+        public override IEnumerable<IOperation> Children
+        {
+            get
+            {
+                if (Target != null)
+                {
+                    yield return Target;
+                }
+                if (WhenNull != null)
```

Comment:
>WhenNull [](start = 20, length = 8)

Can ```WhenNull``` ever be null?

Reply:
Same as above.

---
In reply to: [209382124](https://github.com/dotnet/roslyn/pull/29063#discussion_r209382124) [](ancestors = 209382124)